features/search-replace-export.feature

```diff
@@ -11,7 +11,11 @@ Feature: Search / replace with file export
       """
     And STDOUT should contain:
       """
-      INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`) VALUES ('1', 'siteurl', 'http://example.net', 'yes');
+      INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`) VALUES
+      """
+    And STDOUT should contain:
+      """
+      ('1', 'siteurl', 'http://example.net', 'yes')
```

Comment:
We need a test that multiple chunks get created, and that those multiple chunks are appropriately inserted into the database.